content/en/blog/_posts/2022-12-05-Security-Behavior-Analysis/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2022-12-05
```

Comment:
Date needs to change here and in the filename based on the actual date you publish this date as per SIG Docs schedule